src/neuroconv/datainterfaces/ecephys/baselfpextractorinterface.py

```diff
@@ -36,7 +36,7 @@ def run_conversion(
         overwrite: bool = False,
         stub_test: bool = False,
         starting_time: Optional[float] = None,
-        use_times: bool = False,
+        use_times: bool = False,  # To-do to remove, deprecation
```

Comment:
Not that critical, but using `#TODO` instead will lint it using standard tools like Pylint. Does `# To-do` lint in yours and if so what type are you using?